DOCS/canonical-identity-platform-tutorial-deploy-identity-bundle.md

```diff
@@ -1,191 +1,209 @@
-# Connect the Canonical Identity Platform with an external Identity Provider
+App       Version  Status   Scale  Charm                                Channel        Rev  Address         Exposed  Message
+hydra     v2.3.0   active       1  hydra                                latest/stable  362  10.152.183.80   no       
+kratos    v1.3.1   active       1  kratos                               latest/stable  527  10.152.183.36   no       
+login-ui  0.20.0   active       1  identity-platform-login-ui-operator  latest/stable  166  10.152.183.192  no       
 
-> This part of the tutorial assumes that:
-> - you are in possession of a Microsoft Entra ID tenant and a user with sufficient permissions to provision a client application
-> - you have a client registered in Entra ID; you need to know the `client_id`, `client_secret` and `tenant_id`.
+Unit         Workload  Agent  Address      Ports  Message
+hydra/0*     active    idle   10.1.57.88          
+kratos/0*    active    idle   10.1.57.102         
+login-ui/0*  active    idle   10.1.57.92          
 
-> See more: [Microsoft | Register a new application](https://learn.microsoft.com/en-us/azure/healthcare-apis/register-application#register-a-new-application), [Microsoft | Register a new application > Certificates & Secrets](https://learn.microsoft.com/en-us/azure/healthcare-apis/register-application#certificates--secrets) and [Microsoft | How to find your Microsoft Entra tenant ID](https://learn.microsoft.com/en-us/entra/fundamentals/how-to-find-tenant)
+Offer              Application  Charm   Rev  Connected  Endpoint     Interface    Role
+kratos-info-offer  kratos       kratos  527  0/0        kratos-info  kratos_info  provider
+oauth-offer        hydra        hydra   362  0/0        oauth        oauth        provider
+```
 
+# Use the built-in Identity Provider
 
-[note]
+The Identity Platform comes with a built-in identity provider,
+allowing you to manage user accounts internally without relying on a third-party service.
+You can also let your users authenticate both with external providers and the local one.
 
-If you don't have a Microsoft Entra ID tenant, you can try out the Identity Platform in identity provider mode.
-To do that, proceed to the next tutorial section.
+The local identity provider feature is enabled by default in Charmed Kratos.
 
-[/note]
+At this stage, you can create your personal admin account:
 
-In this tutorial we are going to connect the Identity Platform with Microsoft Entra ID.
+```shell
+juju run kratos/0 create-admin-account email=<your-email> username=<username>
+```
 
-First, we need to inform Entra ID of our deployment’s `redirect_uri`. Get the Kratos base URL:
+After the account is created, use the provided password reset link to set your password and complete the setup.
 
-```
-juju run traefik-public/0 show-proxied-endpoints | yq '.proxied-endpoints' | jq '.kratos.url'
-```
+Congratulations, you're all set to use the built-in identity provider to sign in.
```

Comment:
Correct me if I'm wrong, but doesn't SSO require the external identity provider? If the tutorial wants to demonstrate SSO, then you should just focus on that and maybe highlight the built-in option merely in a note, e.g., "Did you know? The Canonical Identity Platform also has a built-in provider, for cases where you want to handle identity locally."

Reply:
When Identity Platform acts as an identity provider itself, it's still SSO, just with a locally managed identity source.
You sign in once to the Platform, regardless of the authentication method, and this allows you to access multiple applications or systems using the same session.